tests/test_for_support/test_for_helpers.py

```diff
@@ -79,7 +168,56 @@ def proton_pump(base):
 
 
 @register_with(MODEL_REGISTRY)
-def phosphotransferase_system(base):
+def proton_pump_annotations(base):
+    """Provide a model with an ABC proton pump reaction."""
+    atp = cobra.Metabolite("atp_c", formula='C10H12N5O13P3', compartment="c")
+    adp = cobra.Metabolite("adp_c", formula='C10H12N5O10P2', compartment="c")
+    h_c = cobra.Metabolite("h_c", formula='H', compartment="c")
+    pi = cobra.Metabolite("pi_c", formula='HO4P', compartment="c")
+    h2o = cobra.Metabolite("h2o_c", formula='H2O', compartment="c")
+    h_p = cobra.Metabolite("h_p", formula='H', compartment="p")
+
+    atp.annotation["bigg.metabolite"] = "atp"
+    atp.annotation["biocyc"] = ["META:ATP", "META:CPD0-1634"]
+    atp.annotation["metanetx.chemical"] = "MNXM13"
+    atp.annotation["seed.compound"] = "cpd00011"
+
+    adp.annotation["bigg.metabolite"] = "adp"
+    adp.annotation["biocyc"] = ["META:ADP", "META:CPD0-1651"]
+    adp.annotation["metanetx.chemical"] = "MNXM7"
+    adp.annotation["seed.compound"] = "cpd00008"
+
```

Comment:
Here too, it is really not necessary to blow up the code like that. Let's keep it concise wherever possible.